### DISCUSSION TITLE
Feature: search for blog content

### DISCUSSION THREAD

**Discussion by lirantal** [arthelokyo/astrowind#52] [Ideas]
Can we integrate some sort of search functionality for blog contents across the website?

**Comment by prototypa** [comment]
Hello @lirantal 

It's a good idea, it's already been asked here: https://github.com/onwidget/astrowind/discussions/27

In the next iteration I will work on it.

**Reply by lirantal** [reply]
Ahh missed that. Thanks!

**Comment by quetzalsly** [comment]
hey @lirantal I just saw your personal website now has a search feature. It looks great!
Can you please share how you did it?
Thanks

**Reply by lirantal** [reply]
Thanks! I documented it here: https://lirantal.com/blog/2023-01-01_-how_to_add_client-side_search_to_your_astro_blog_static_website/
